includes/payments/actions.php

```diff
@@ -171,8 +171,7 @@ function edd_undo_purchase_on_refund( $payment_id, $new_status, $old_status ) {
 
 	if( $customer_id ) {
 
-		$customer = new EDD_Customer( $customer_id );
-		$customer->decrement_stats( $customer_id, $amount );
+		EDD()->customers->decrement_stats( $customer_id, $amount );
```

Comment:
This is what we use literally every other place in EDD to do this (at least 6 other places). What's so bad about consistency?